packages/components/src/components/contract-card/contract-card-items/accumulator-card-body.tsx

```diff
@@ -26,11 +48,16 @@ const AccumulatorCardBody = ({
     setCurrentFocus,
     status,
     is_positions,
-}) => {
+}: TAccumulatorCardBody) => {
     const { buy_price, profit, limit_order, sell_price } = contract_info;
     const { take_profit } = getLimitOrderAmount(contract_update || limit_order);
     const is_valid_to_sell = isValidToSell(contract_info);
     const { CURRENT_STAKE, STAKE, TAKE_PROFIT, TOTAL_PROFIT_LOSS } = getCardLabels();
+    let is_won, is_loss;
+    if (profit) {
+        is_won = +profit > 0;
```

Comment:
if `is_won` always has an opposite value to `is_loss`, maybe it'll make sense to use only one of them?

Reply:
In TContractInfo profit is `profit?: number;`, so maybe there is no need in "+"?

Reply:
no, i think `open` is also one of the conditions. Also, I didnt want to change the logic while doing typescript conversion

Reply:
mmmm not sure. `+profit` was in there so i kept it the same way.

Reply:
@henry-deriv can we make them const since we don't overwrite them anywhere :) ?
```
const is_loss = !!profit && +profit < 0;
const is_won = !!profit && +profit > 0;
```